test/unit/int_algolia/scripts/algolia/model/algoliaLocalizedProduct.test.js

```diff
@@ -504,3 +511,62 @@ describe('algoliaLocalizedProduct overriding custom attributes', function () {
         expect(new AlgoliaLocalizedProductModel({ product: product, locale: 'fr', attributeList: ['algoliaTest', 'custom.displaySize'], baseModel: baseModel})).toEqual(expectedProductModel);
     });
 });
+describe('productFilter.isIncludeOutOfStock', () => {
```

Comment:
Is there a reason to unit test methods of an external helper here?
If not, please create a `productFilter.test.js` file, which is the Jest convention